### ISSUE TITLE
Crash when adding new comment

### ISSUE THREAD

**Issue by vecheslav255** [nextcloud/android#12353]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Folder properties -> Details -> Events -> New comment -> Input -> Crash.
Repeats on any folder

### Expected behaviour

There should be a new comment

### Actual behaviour

nextcloud app crash

### Android version

11

### Device brand and model

Xiaomi

### Stock or custom OS?

Stock

### Nextcloud android app version

Redmi Note 8T

### Nextcloud server version

Nextcloud Hub 7 (28.0.1)

### Using a reverse proxy?

No

### Cause of error
```java
Exception in thread "AsyncTask #2" java.lang.RuntimeException: An error occurred while executing doInBackground()
    at android.os.AsyncTask$4.done(AsyncTask.java:415)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
    at java.util.concurrent.FutureTask.run(FutureTask.java:271)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    at java.lang.Thread.run(Thread.java:923)
Caused by: Exception in thread "AsyncTask #2" java.lang.IllegalArgumentException: Trying to execute a remote operation with a NULL OwnCloudClient
    at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:201)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:488)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:467)
    at android.os.AsyncTask$3.call(AsyncTask.java:394)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    at java.lang.Thread.run(Thread.java:923)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30270090`
* Build flavor: `gplay`

### Device information
* Brand: `xiaomi`
* Device: `willow`
* Model: `Redmi Note 8T`
* Id: `RKQ1.201004.002`
* Product: `willow`

### Firmware
* SDK: `30`
* Release: `11`
* Incremental: `V12.5.6.0.RCXMIXM`

### Server error logs

```bash
None
```

### Additional information

none

**Comment by vecheslav255** [comment]
![Screenshot_2024-01-10-12-49-02-835_com nextcloud client](https://github.com/nextcloud/android/assets/63278365/9593291b-8d76-4836-af00-ded6d8d93a05)

**Comment by svenb1234** [comment]
### Cause of error
```java
Exception in thread "AsyncTask #3" java.lang.RuntimeException: An error occurred while executing doInBackground()
    at android.os.AsyncTask$4.done(AsyncTask.java:415)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
    at java.util.concurrent.FutureTask.run(FutureTask.java:269)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
    at java.lang.Thread.run(Thread.java:1012)
Caused by: Exception in thread "AsyncTask #3" java.lang.IllegalArgumentException: Trying to execute a remote operation with a NULL OwnCloudClient
    at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:201)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:489)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:468)
    at android.os.AsyncTask$3.call(AsyncTask.java:394)
    at java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30280090`
* Build flavor: `gplay`

### Device information
* Brand: `google`
* Device: `oriole`
* Model: `Pixel 6`
* Id: `UQ1A.240205.002`
* Product: `oriole`

### Firmware
* SDK: `34`
* Release: `14`
* Incremental: `11224170`

**Comment by Wurftaube** [comment]
Same issue

### Cause of error
```java
Exception in thread "AsyncTask #26" java.lang.RuntimeException: An error occurred while executing doInBackground()
    at android.os.AsyncTask$4.done(AsyncTask.java:399)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
    at java.util.concurrent.FutureTask.run(FutureTask.java:271)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:289)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    at java.lang.Thread.run(Thread.java:919)
Caused by: Exception in thread "AsyncTask #26" java.lang.IllegalArgumentException: Trying to execute a remote operation with a NULL OwnCloudClient
    at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:201)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:489)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:468)
    at android.os.AsyncTask$3.call(AsyncTask.java:378)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:289)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    at java.lang.Thread.run(Thread.java:919)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30280090`
* Build flavor: `generic`

### Device information
* Brand: `samsung`
* Device: `crownlte`
* Model: `SM-N960F`
* Id: `QP1A.190711.020`
* Product: `crownltexx`

### Firmware
* SDK: `29`
* Release: `10`
* Incremental: `N960FXXU9FVH1`

**Comment by JoergTheHipp** [comment]
This bug is still existing, is there planned date to fix it? ( I'm having a different Hardware Samsung S20 FE)

**Comment by joshtrichards** [comment]
Fixed in #12583
